client/src/inside/components/mar/mar-entity.js

```diff
@@ -1,12 +1,12 @@
 import { validTimeStr, validDayStr } from '../../../helpers/ehr-utils'
-import PeriodDefs from './period-defs'
-import MedOrder from './med-entity'
+import MedOrder from './med-order'
 
 
 const ERR_WHO = 'Must provide who administered'
 const ERR_WHEN = 'Time must be 24:00 hour format'
 const ERR_DAY = 'Day must be one of 0,1,2,3,...'
 const ERR_EMPTY_MEDS = 'Medication list can not be empty'
+// TODO see if we need to validate scheduled time
```

Comment:
Is this TODO still a todo? If yes then we need to explain it more.  Ideally make a github issue and place the issue number into the code in this todo comment

Reply:
I think so, I believe you have added it. Could you please provide a few more details so that I can create the issue? Thank you!